### PR TITLE
Assessment list and footer

### PR DESCRIPTION
**Pull request by rickymetz**
Implements new design for assessment list page.
* Refactored `assessment-list.html`
  * Stripped out unnecessary bootstrap column code in favour of vanilla css
  * Removed extraneous classes
* Completely rewrote `assessments.sass` to eliminate useless code (661KB

screenshot 2016-03-10 15 01 05

### CODE REVIEW

File: app/assets/javascripts/client/views/directives/start_assessment.html [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
-          <i class="fa fa-plus"></i>
-        </div>
-        <div class="col-sm-11 col-md-11 heading">
-          <h1>{{text()}}</h1>
+        <span class="fa-stack">
+          <i class="fa fa-circle fa-stack-2x"></i>
+          <i class="fa fa-plus fa-stack-1x"></i>
+        </span>
+        <div class="heading">
+          <a>{{text()}}</a>
```
Comment (MakotoTheKnight) [inline app/assets/javascripts/client/views/directives/start_assessment.html]: This anchor tag interferes with the parent anchor tag which drives the modal. What that results in is modal not opening when the text is clicked, but only when the icon is clicked.

Removing this anchor tag would suffice.

Comment (linqiu) [inline app/assets/javascripts/client/views/directives/start_assessment.html]: @rickymetz why is this text file in an anchor tag that doesn't do anything? :trollface:

Comment (rickymetz) [inline app/assets/javascripts/client/views/directives/start_assessment.html]: @linqiu stay in your own repos!

**Comment by MakotoTheKnight** [comment]
As a network partner, this is what I see:

image

The text there seems a bit large in comparison to the other button.

**Comment by rickymetz** [comment]
It is. I noticed this this morning. I have to direct some time into Urban, so I may or may not get to that today. You could merge it, then I could make a smaller separate PR that captures this issue.

**Comment by MakotoTheKnight** [comment]
That's fine with me.  We can go forward with this.

**Comment by rickymetz** [comment]
:+1:

**Pull request merged**